engine/antigravity_engine/hub/ask_pipeline.py

```
@@ -24,10 +24,106 @@
     WorkerEvidence,
 )
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from agents.result import RunResult, RunResultStreaming
 
 logger = logging.getLogger(__name__)
 
 
+async def _run_with_optional_stream(
+    agent: "Agent",
```

Ruff (0.15.11)

[error] Undefined name `Agent`

(F821)